packages/metrics/src/Metrics.ts

```diff
@@ -240,38 +276,37 @@ class Metrics extends Utility implements MetricsInterface {
   /**
    * Add a metric to the metrics buffer.
    *
-   * By default, metrics are buffered and flushed at the end of the Lambda invocation
-   * or when calling {@link Metrics.publishStoredMetrics}.
+   * By default, metrics are buffered and flushed when calling {@link Metrics.publishStoredMetrics | `publishStoredMetrics()`} method,
+   * or at the end of the handler function when using the {@link Metrics.logMetrics | `logMetrics()`} decorator or the Middy.js middleware.
    *
-   * You can add a metric by specifying the metric name, unit, and value. For convenience,
-   * we provide a set of constants for the most common units in {@link MetricUnit}.
+   * Metrics are emitted to standard output in the Amazon CloudWatch EMF (Embedded Metric Format) schema. In AWS Lambda, the logs are
+   * automatically picked up by CloudWatch logs and processed asynchronously.
```

Comment:
```suggestion
   * Metrics are emitted to standard output in the Amazon CloudWatch EMF (Embedded Metric Format) schema.
```

Customers can remove CloudWatch permissions, can subscribe logs to api or other. So, it's better to keep that this utility just emits logs in this format.